README.rst

```diff
@@ -0,0 +1,105 @@
+image here
+image target here
+
+image here
+image target here
+
+
+PDFmorph
+========================================================================
+add description, sample below
+Code package for morphin' atomic pair distribution functions.
+
+PDFmorph is a user-friendly software package. [more info here]
+
+
```

Comment:
probably need a license statement in here somewhere.

Also a section that lists the developer team.   CJ may have a way to build that automatically using Rever.

Reply:
@CJ-Wright and I are working on it!

Reply:
Please fix license at least.  It is a copy-paste-edit exercise and doesn't need CJ.